### ISSUE TITLE
run "make install" failed

### ISSUE THREAD

**Issue by luckypoem** [racket/racket#660]
hi.

```
wget http://mirror.racket-lang.org/installers/6.0.1/racket-6.0.1-src.tgz
tar zxvf racket-6.0.1-src.tgz
cd racket-6.0.1
cd src
./configure
make
make install

---------------------------------------------------------------
as3:~/racket-6.0.1/src# make install
make install-3m
make[1]: Entering directory `/root/racket-6.0.1/src'
make install-3m-common
make[2]: Entering directory `/root/racket-6.0.1/src'
make install-common-first
make[3]: Entering directory `/root/racket-6.0.1/src'
mkdir -p "/root/racket-6.0.1/bin" "/root/racket-6.0.1/collects" "/root/racket-6.0.1/doc" "/root/racket-6.0.1/lib" "/root/racket-6.0.1/include" "/root/racket-6.0.1/lib" "/root/racket-6.0.1/share" "/root/racket-6.0.1/etc" "/root/racket-6.0.1/share/applications" "/root/racket-6.0.1/man"
make[3]: Leaving directory `/root/racket-6.0.1/src'
cd racket; make install-3m
make[3]: Entering directory `/root/racket-6.0.1/src/racket'
make headers
make[4]: Entering directory `/root/racket-6.0.1/src/racket'
./racketcgc -cqu ./mkincludes.rkt "/root/racket-6.0.1/include" "." .
Making /root/racket-6.0.1/include
Done.
make[4]: Leaving directory `/root/racket-6.0.1/src/racket'
make unix-install
make[4]: Entering directory `/root/racket-6.0.1/src/racket'
cd ..; rm -f "/root/racket-6.0.1/bin/racketcgc"
cd ..; rm -f "/root/racket-6.0.1/bin/racket"
cd ..; cp racket/starter "/root/racket-6.0.1/lib/starter"
cp ./dynsrc/starter-sh .
cd ..; cp racket/starter-sh "/root/racket-6.0.1/lib/starter-sh"
cd ..; strip "/root/racket-6.0.1/lib/starter"
./racketcgc -cu "./collects-path.rkt" "/root/racket-6.0.1/lib/starter" ../collects ../etc
cd ..; echo 'CC=gcc' > "/root/racket-6.0.1/lib/buildinfo"
cd ..; echo 'CFLAGS=-g -O2  -Wall     -pthread  ' >> "/root/racket-6.0.1/lib/buildinfo"
cd ..; echo 'OPTIONS=' >> "/root/racket-6.0.1/lib/buildinfo"
cd ..; echo 'MZOPTIONS= -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64' >> "/root/racket-6.0.1/lib/buildinfo"
cd ..; echo "LDFLAGS= -pthread" >> "/root/racket-6.0.1/lib/buildinfo"
cd ..; echo 'LIBS=-ldl -lm  -ldl -lm -rdynamic -lffi   ' >> "/root/racket-6.0.1/lib/buildinfo"
make[4]: Leaving directory `/root/racket-6.0.1/src/racket'
make unix-install-3m
make[4]: Entering directory `/root/racket-6.0.1/src/racket'
cd ..; cp racket/racket3m "/root/racket-6.0.1/bin/racket"
cd ..; strip "/root/racket-6.0.1/bin/racket"
make unix-no-install-libs-3m
make[5]: Entering directory `/root/racket-6.0.1/src/racket'
:
make[5]: Leaving directory `/root/racket-6.0.1/src/racket'
cd ..; cp racket/mzdyn3m.o "/root/racket-6.0.1/lib/mzdyn3m.o"
./racket3m -cu "./collects-path.rkt" "/root/racket-6.0.1/bin/racket" ../collects ../etc
make[4]: Leaving directory `/root/racket-6.0.1/src/racket'
make unix-install-3m-final
make[4]: Entering directory `/root/racket-6.0.1/src/racket'
:
make[4]: Leaving directory `/root/racket-6.0.1/src/racket'
make[3]: Leaving directory `/root/racket-6.0.1/src/racket'
make install-gracket-3m
make[3]: Entering directory `/root/racket-6.0.1/src'
cd gracket; make install-3m
make[4]: Entering directory `/root/racket-6.0.1/src/gracket'
make install-wx_xt
make[5]: Entering directory `/root/racket-6.0.1/src/gracket'
make install-common
make[6]: Entering directory `/root/racket-6.0.1/src/gracket'
:
make[6]: Leaving directory `/root/racket-6.0.1/src/gracket'
cd ..; rm -f "/root/racket-6.0.1/lib/gracketcgc"
cd ..; rm -f "/root/racket-6.0.1/lib/gracket"
cd ..; echo 'MROPTIONS=' >> "/root/racket-6.0.1/lib/buildinfo"
cd ..; echo "MRLIBS=" >> "/root/racket-6.0.1/lib/buildinfo"
cd ..; echo "MRLDFLAGS=-pthread -L../racket" >> "/root/racket-6.0.1/lib/buildinfo"
cd ..; mkdir -p "/root/racket-6.0.1/lib";
make[5]: Leaving directory `/root/racket-6.0.1/src/gracket'
make install-wx_xt-3m
make[5]: Entering directory `/root/racket-6.0.1/src/gracket'
make install-no-lib-3m-wx_xt
make[6]: Entering directory `/root/racket-6.0.1/src/gracket'
echo "no dynamic libs"
no dynamic libs
make[6]: Leaving directory `/root/racket-6.0.1/src/gracket'
cd ..; cp gracket/gracket3m "/root/racket-6.0.1/lib/gracket"
cd ..; strip "/root/racket-6.0.1/lib/gracket"
../racket/racket3m  -cu "./../racket/collects-path.rkt" "/root/racket-6.0.1/lib/gracket" ../collects ../etc
make[5]: Leaving directory `/root/racket-6.0.1/src/gracket'
make install-wx_xt-3m-final
make[5]: Entering directory `/root/racket-6.0.1/src/gracket'
make[5]: Leaving directory `/root/racket-6.0.1/src/gracket'
make[4]: Leaving directory `/root/racket-6.0.1/src/gracket'
make[3]: Leaving directory `/root/racket-6.0.1/src'
make install-common-middle
make[3]: Entering directory `/root/racket-6.0.1/src'
make no-run
make[4]: Entering directory `/root/racket-6.0.1/src'
:
make[4]: Leaving directory `/root/racket-6.0.1/src'
make install-gracket-post-collects
make[4]: Entering directory `/root/racket-6.0.1/src'
cd gracket; make install-post-collects
make[5]: Entering directory `/root/racket-6.0.1/src/gracket'
make install-wx_xt-post-collects
make[6]: Entering directory `/root/racket-6.0.1/src/gracket'
:
make[6]: Leaving directory `/root/racket-6.0.1/src/gracket'
make[5]: Leaving directory `/root/racket-6.0.1/src/gracket'
make[4]: Leaving directory `/root/racket-6.0.1/src'
make lib-finish
make[4]: Entering directory `/root/racket-6.0.1/src'
if [ "" = "" ]; then \
          echo "/root/racket-6.0.1/lib"; \
        fi
/root/racket-6.0.1/lib
make[4]: Leaving directory `/root/racket-6.0.1/src'
make[3]: Leaving directory `/root/racket-6.0.1/src'
make[2]: Leaving directory `/root/racket-6.0.1/src'
racket/racket3m -X "/root/racket-6.0.1/collects" -G "/root/racket-6.0.1/etc"  -N "raco" -l- setup  --no-user
raco setup: bootstrapping from source...
raco setup: version: 6.0.1 [3m]
raco setup: installation name: 6.0.1
raco setup: variants: 3m
raco setup: main collects: /root/racket-6.0.1/collects
raco setup: collects paths:
raco setup:   /root/racket-6.0.1/collects
raco setup: main pkgs: /root/racket-6.0.1/share/pkgs
raco setup: pkgs paths:
raco setup:   /root/racket-6.0.1/share/pkgs
raco setup:   /root/.racket/6.0.1/pkgs
raco setup: links files:
raco setup:   /root/racket-6.0.1/share/links.rktd
raco setup:   /root/.racket/6.0.1/links.rktd
raco setup: main docs: /root/racket-6.0.1/doc
raco setup: --- updating info-domain tables ---
raco setup: updating: <collects>/info-domain/compiled/cache.rktd
raco setup: updating: /root/racket-6.0.1/share/info-cache.rktd
raco setup: --- pre-installing collections ---
raco setup: --- installing foreign libraries ---
raco setup: --- installing shared files ---
raco setup: installing: shared file <share>/slatex.sty
raco setup: installing: shared file <share>/slatex.py
raco setup: --- compiling collections ---
raco setup: making: <collects>/racket
raco setup:  in <collects>/racket
raco setup:  in <collects>/syntax
raco setup:  in <collects>/racket/private
raco setup: --- parallel build using 2 jobs ---
raco setup: 1 making: <collects>/compiler
raco setup: 0 making: <pkgs>/cext-lib/compiler
raco setup: 0 making: <pkgs>/cext-lib/compiler/commands
raco setup: 1 making: <collects>/compiler/private
raco setup: 1 making: <pkgs>/compiler-lib/compiler
raco setup: 0 making: <pkgs>/mzscheme-lib/compiler
raco setup: 0 making: <collects>/raco
make[1]: *** [install-3m] Killed
make[1]: Leaving directory `/root/racket-6.0.1/src'
make: *** [install] Error 2
as3:~/racket-6.0.1/src#
```

how to fix it?
tks a lot

**Comment by harrisi** [comment]
I had some trouble installing 6.0.1 (on Fedora, which is relevant below), but 6.4 seems fine.

On @mflatt's note, I did run into a similar error as this (as well as #769) due to running out of either memory or storage space (I ran into both). In both cases, Racket itself built fine but raco wasn't fully made. However, increasing both the memory and the storage resolved errors. Without more information on OS and racket version, I don't know how much more could be done with this issue.
